app/components/add-new-goal/new-goal-avatars.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const Avatars = () => {
+
+  const pathToAvatars = `./app/public/images/add-new-goal/`;
+
+  // could refactor to get avatars from state in case we want to load from DB?
+  const availableAvatars = [
+    'sprout',
+    'pepper',
+    'pumpkin',
+    'flower',
+  ];
+
+  const createAvatarRow = arr =>
+  arr.map(av => (
+    <div
+      className="newGoal_avatarBox"
+      key={`avatar_${av}`}
+      id={`avatar_${av}`}
+      >
+      <img src={`${pathToAvatars}${av}.png`} />
```

Comment:
You could make pathToAvatars a function like
`<img src={ pathToAvatars(av)} />`